lib/index.js

```diff
@@ -9,6 +9,10 @@ module.exports = function (options) {
   let base_url = options.base_url || false;
   let response_code = options.response_code || false;
 
+  function escapePipes(str) {
+    return str.replace(/\|/g, "\\|");
```

Comment:
since this might get call for every single request, let's declare the regex outside of the function. What do you think? (it is a small performance suggestion).

Reply:
also to be super safe, let's make sure `str` does exist `str && str.replace(pattern, '\\|');`